enjarify.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
```

Comment:
Do we really want to include this file in versioning ?
I would rather have a dedicated tools subdirectory, and store all of them as a zip file